### PR TITLE
implemeted tests for the app

### PR DESCRIPTION
**Pull request by BenM4k**
In the present update, I :

implemented the design following these [guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))
implemented a home page containing categories to be filtered. Each category has numeric data coming from the API, and clicking on a category brings you to its detail page
implemented a detail page containing more details about the category. It has a back bouton that brings you home
implemented tests for the whole application
For more details, take a look at the official [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md)

### CODE REVIEW

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
👍🏽 Linters installed and the PR passes the checks ✔️ 
👍🏽 You have recorded a video and you presented your projects correctly, except for just a few details you missed from it ✔️ 
👍🏽 The PR has a description and a title ✔️ 
👍🏽 The PR follows the correct git/github flow ✔️ 
👍🏽 The project is deployed and it renders correctly ✔️ 

## Required Changes ♻️
_Check the comments under the review._
- [ ] From your readme file, the command you said to run test cases is wrong, in order to run test cases, the right command to run is `npm test` not linters command
Screenshot 2023-08-03 at 19 34 44

- [ ] From the video, you have missed one of the required details which is: `The video contains a highlight about a piece of the code the student finds interesting or is proud of making.`

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- [ ] You could put a link to the design to which you are giving credit in your readme file

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @BenM4k, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy


## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally


## Suggested changes
_Check the comments under the review._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -93,13 +93,6 @@ To run the project, execute the following command:
...
 
-### Run tests
-
-To run tests, run the following command:
-
-- npx stylelint "**/*.{css,scss}"
-- npx hint .
-
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
[the remark this answers is not in the document]
Comment (Abenezer-Tilahun) [reply README.md:103]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @BenM4k 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

**Pull request merged**